console/console-init/ui/src/modules/msg-and-iot/dailogs/containers/CreateProjectContainer/CreateProjectContainer.tsx

```diff
@@ -75,12 +88,30 @@ const CreateProjectContainer: React.FunctionComponent = () => {
           }
         }
       };
-      const data: any = await setQueryVariables(variables);
-      console.log(data);
+      setQueryVariables(variables);
       resetForm();
     }
   };
 
+  const { loading, data } = useQuery<INamespaces>(RETURN_NAMESPACES);
+  if (loading) return <Loading />;
+
+  const { namespaces } = data || {
+    namespaces: []
+  };
+
+  const namespaceOptions = namespaces.map(namespace => {
```

Comment:
can you try to use function  as below.
//prepareNamespaceOptions or getNamespaceOptions
const prepareNamespaceOptions=()=>{
  namespaces.map(namespace => {
  const {metadata}=namespace;
    return {
      value: metadata.name,
      label: metadata.name
    };
 }
});